yarn.lock

```diff
@@ -3132,24 +3140,50 @@
   dependencies:
     path-to-regexp "6.1.0"
 
+"@openmrs/esm-breadcrumbs@^3.1.15-pre.749":
+  version "3.1.15-pre.749"
+  resolved "https://registry.yarnpkg.com/@openmrs/esm-breadcrumbs/-/esm-breadcrumbs-3.1.15-pre.749.tgz#241ac3a75adb7cd1e94ff249c0c6ab881af223af"
+  integrity sha512-+JslX8U6jGh/4gskb7OFWTlKVpNBn5Dd306nS70W/EdJKE4pSPtcYQ0wPQxwWDUaRXDYBC5RwIaIPX+oseq7eA==
+  dependencies:
+    path-to-regexp "6.1.0"
+
 "@openmrs/esm-config@^3.1.14-pre.676":
   version "3.1.14-pre.676"
   resolved "https://registry.yarnpkg.com/@openmrs/esm-config/-/esm-config-3.1.14-pre.676.tgz#41f94c9e7b59f7738c9602133782e289a6640894"
   integrity sha512-BCevDYJ3N9B19UJBxKsauOsJ9HGKzaiLEYLx6GQlSVpZ+yN3yYaXPc+AWCf5LN5TjzlO08FKGrNVj+yZzzqJ3g==
   dependencies:
     ramda "^0.26.1"
 
+"@openmrs/esm-config@^3.1.15-pre.749":
```

Comment:
I'm a little concerned that the yarn lockfile now shows version 676 and version 749 of all the openmrs dependencies. But maybe it's fine, I don't really know.